pkg/cmdline/cmdline.go

```diff
@@ -84,32 +83,64 @@ func printableTypeName(typ reflect.Type) string {
 	} else if typ.Kind() == reflect.Map {
 		return fmt.Sprintf("JSON dict of %s to %s", printableTypeName(typ.Key()), printableTypeName(typ.Elem()))
 	} else if typ.Kind() == reflect.Slice {
+		if typ.Elem() == reflect.TypeOf("") {
+			return fmt.Sprintf("%s (may be repeated)", typ.String())
```

Comment:
does this mean comma-separated?
e.g.
`dnsname: foo.com,foo2.com`

Reply:
No, it means you can give the option more than once, adding a single string each time, and the cmdlibrary will build a slice of strings that it passes to the app.  So you can do `receptor --cert-makereq [...] dnsname=foo.com dnsname=bar.com`